registry/storage/driver/gcs/gcs.go

```diff
@@ -439,19 +410,18 @@ func (w *writer) Close() error {
 	return nil
 }
 
-func putContentsClose(wc *storage.Writer, contents []byte) error {
-	size := len(contents)
-	var nn int
-	var err error
-	for nn < size {
-		var n int
-		n, err = wc.Write(contents[nn:size])
-		nn += n
-		if err != nil {
-			break
-		}
+func (d *driver) flushContent(ctx context.Context, obj *storage.ObjectHandle, content []byte, contentType string, metadata map[string]string) error {
```

Comment:
I can't accept this suggestion unless you also make suggestions for all the code that calls this func with the new name, but happy to change.

Reply:
I don't need the attribution. I only really use GH suggestions because they render coloured diffs automatically. You'd be faster renaming the method using IDE refactor tools than I would by spamming GH suggestions

Reply:
Done: https://github.com/distribution/distribution/pull/4120/commits/ce47da9d98719bdfcc076ce4caf272cf4084ec03